app.py

```
@@ -21,13 +19,19 @@
 @app.route("/api/<title>", methods=["GET"])
 def get_title(title) -> str:
     # ---
+    try:
+        from make2 import event
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)